refinery/ui/source/js/tool-launch/partials/tool-launch-warning.html

```diff
@@ -0,0 +1,10 @@
+<div
+  id="tool-launch-warning"
+  class="alert alert-warning"
+  ng-if="$ctrl.userIsAnonymous()">
+  You will need to
+  <strong>
+    <a href="/accounts/register/">register for an account</a>
```

Comment:
Perhaps just "Login to launch visualizations"? In a sense registration is neither necessary nor sufficient: There could be a couple days between registration and approval, and unless I'm actually logged in, registration won't do me much good. And perhaps "tools" -> "visualizations": They still in all likelihood won't be able to launch workflows, and don't want to get their hopes up.